src/DotLiquid.Tests/HashTests.cs

```diff
@@ -117,5 +118,50 @@ public void TestUpperTwoScenarioWithSameClass()
             IncludeBaseClassPropertiesOrNot(true);
         }
         #endregion
+
+        [Test]
+        public void TestDefaultConstructor()
```

Comment:
TestDefaultValueConstructor might be more accurate.

Reply:
Agreed.